specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -5173,7 +5173,11 @@
           "description": "Expression type.",
           "enum": [
             "Expression"
-          ]
+          ],
+          "x-ms-enum": {
+            "name": "ExpressionType",
+            "modelAsString": false
```

Comment:
Please use `modelAsString`: true, which could allow more values to add to this enum.

Reply:
Thanks, updated